packages/cli/src/License.ts

```diff
@@ -65,6 +69,7 @@ export class License {
 				loadCertStr: async () => this.loadCertStr(),
 				saveCertStr,
 				deviceFingerprint: () => this.instanceSettings.instanceId,
+				collectUsageMetrics,
```

Comment:
I'm guessing this isn't going to be called frequently enough to try to optimize the DB calls made here?

Reply:
`collectUsageMetrics()` is only called prior license renewal, which happens once every 10 days by default. It is also triggered when n8n starts up and when the _usage and plan_ page is opened, but probably not often enough to optimize DB calls 😄